near-metadata/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "near-metadata"
+version = "0.1.0"
+authors = ["Sougata Bhattacharjee <sb@shardlabs.io>"]
```

Comment:
Please rename to 
https://github.com/Shard-Labs/rainbow-non-fungible-token-connector/blob/reverse-connector/nft-token-factory/Cargo.toml#L4